source/manuals/access-aws-accounts.html.md.erb

```diff
@@ -11,20 +11,17 @@ account_list:
 
 # <%= current_page.data.title %>
 
-GDS manage a number of Amazon Web Services (AWS) accounts.
-User accounts are managed centrally in a "base account" from which users
-can access the target accounts through a process called "assuming roles".
+GDS manages a number of [Amazon Web Services (AWS)][] accounts. User accounts are managed centrally by Reliability Engineering with a base GDS  account. Users must sign into the base account through which they can access their own AWS accounts through a [process called assuming roles][].
 
-## Adding new users to GDS' AWS accounts
+## Add new users to GDS AWS accounts
 
-Currently new joiners have no access to AWS by default. To grant a user access
-to AWS they need to be added to the `gds-users` base account. Once a user is in
-the base account it is up to the owners of the target accounts to grant the
-user access to roles within the account.
+People joining GDS do not automatically have access to AWS.
 
-To add a user to the `gds-users` base account fill in [this form](https://gds-request-an-aws-account.cloudapps.digital/).
+To grant a user access to AWS you'll need to add them to the `gds-users` base account. Once a user is added to the GDS base account it's up to individual teams to grant the user access to any roles they need.
 
-## Accessing the AWS Console
+Fill in the [Request an AWS account form][] to add a user to the `gds-users` base account.
+
+## Access the AWS Console
 
 <ul>
```

Comment:
We had UR feedback about using these drop down widgets, that they're incompatible with screen readers and can break the left-hand nav' in the tech doc format.